File: Per1/Latihan B/c++/main.cpp
#include <iostream>
#include <string>
using namespace std;
#include "football.cpp"
int main (){
    tim sepakbola;
    int n, tahunI;
    string namaI, negaraI, pelatihI, pemainI; 
    cout << "Masukkan Nama tim: ";
    cin >> namaI;
    sepakbola.setNama(namaI);
    cout << "Masukkan Negara tim: ";
    cin >> negaraI;
    sepakbola.setNegara(negaraI);
    cout << "Masukkan Tahun terbentuknya tim: ";
    cin >> tahunI;
    sepakbola.setTahun(tahunI);
    cout << "Masukkan Pelatih tim: ";
    cin >> pelatihI;
    sepakbola.setPelatih(pelatihI);
    cout << "Masukkan banyaknya Pemain yang ingin dimasukkan: " ;
    cin >> n;
    tim sepakbola2[n];
    for(int i=0;i<n;i++){
        cout << "Masukkan Nama Pemain: " << i+1 << ". ";
        cin >> pemainI;
        sepakbola2[i].setPemain(pemainI);
    }
    cout << "\nNama Tim: " << sepakbola.getNama() << endl;
    cout << "Negara Tim: " << sepakbola.getNegara() << endl;
    cout << "Tahun Terbentuknya Tim: "  << sepakbola.getTahun() << endl;
    cout << "Pelatih Tim: " << sepakbola.getPelatih() << endl;
    cout << "Pemain Tim:\n";
    for(int i = 0;i<n;i++){
        cout << (i+1) << ". " << sepakbola2[i].getPemain() << endl;
    }
    return 0;
}

File: Per1/Latihan B/Py/main.py
from tim import tim

sepakbola = tim()

print("Masukkan Nama Tim: ")
namaI = str(input())
sepakbola.setNama(namaI)
print("Masukkan Negara Tim: ")
negaraI = str(input())
sepakbola.setNegara(negaraI)
print("Masukkan Tahun Tim: ")
tahunI = int(input())
sepakbola.setTahun(tahunI)
print("Masukkan Pelatih tim")
pelatihI = str(input())
sepakbola.setPelatih(pelatihI)
print("Masukkan Banyaknya Pemain Tim: ")
n = int(input())
sepakbola2 = [tim() for i in range(n)]
for i in range(n):
    print("Masukkan Nama Pemain Tim: ")
    pemainI = str(input())
    sepakbola2[i].setPemain(pemainI)
print("")
print("Nama Tim: " + str(sepakbola.getNama()))
print("Negara Tim: " + str(sepakbola.getNegara()))
print("Tahun Tim: " + str(sepakbola.getTahun()))
print("Pelatih Tim: " + str(sepakbola.getPelatih()))
print("Nama Pemain Tim: ")
for i in range(n):
    print(sepakbola2[i].getPemain())